tests/Feature/QuizTest.php

```diff
@@ -23,4 +24,21 @@ public function test_displayAllQuizzes_success(): void
             });
         });
     }
+
+    public function test_addNewQuiz_Success(): void
+    {
+        Quiz::factory()->create();
+        $testData = [
+            'name' => "dave",
+            'description' => "hey",
+        ];
+
+        $response = $this->postJson('/api/quizzes', $testData);
+
+        $response->assertStatus(201)
+            ->assertJson(function (AssertableJson $json) {
+                $json->hasAll(['message']);
+            });
+        $this->assertDatabaseHas('quizzes', $testData);
+    }
```

Comment:
Can you think of any other tests you might want to do now that you're validating user inputs and adding stuff to the database?